### ISSUE TITLE
Need option to re-reverse enginer selected entities without removing all others

### ISSUE THREAD

**Issue by netbitshift** [ErikEJ/EFCorePowerTools#246]
Describe what is not working as expected.

### Steps to reproduce
- Select Db to perform RE (reverse engineer)
- Select all entities, then ok
Notice all the entities generated
Now, go back and perform the same steps but only select 2-3 entities that you want re-generated.  They generate, but all others are removed.

How can one selectively re-reverse engineer a few tables without also blowing away all the others?  My workflow involves making db schema changes on a fairly routine basis (every other project or so), and some of my table entities have to get custom data annotations applied.  Hence, Id rather not have to re-do those or manage the changes with my source control history to get the annotations back in place.   Maybe Im doing something wrong?

**Comment by ErikEJ** [comment]
All the generated files are partial, but maybe that does not help with Data annotations?

**Comment by ErikEJ** [comment]
Ping?

**Comment by ErikEJ** [comment]
Closing as by design

**Comment by netbitshift** [comment]
Sorry for the errr... late response.  Guess I missed this one.  Not sure what you meant by reference to the Data annotations.  My typical workflow will typically involve adding/updating a few tables.  There close to 100 tables in the Db, with a handful of them I have added custom attributes on the Data annotations, eg.
[Required] to [Required(AllowEmptyStrings = true)]

If I simply follow the Reverse eng path, I need to select the tables to be generated, which is used to recreate the db model/context.  I get that, but I suppose what Im wondering is if there is a way to reverse eng only a handful of tables (to reflect the schema changes in the Db for those tables), rather than all or nothing.  If I do the latter, then I loose those custom annotations that were added and they must be added back manually. Granted its only a handful, but it gets me out of the UI workflow.
Hope that helps.

Another suggestion possibly, is to allow custom generation of a handful of entities, where I could specify those custom annotation attributes?

Thanks again, and love the tool!
